src/response/records.ts

```diff
@@ -361,6 +361,111 @@ export interface Disposition {
   readonly ruleLabel?: string;
 }
 
+/**
+ * The risk score reason for the multiplier.
+ *
+ * This class provides both a machine-readable code and a human-readable
+ * explanation of the reason for the risk score, see
+ * {@link https://dev.maxmind.com/minfraud/api-documentation/responses/#schema--response--risk-score-reason--multiplier-reason}.
```

Comment:
This is a very long link to appear inline in the docs. Maybe something like:

```suggestion
 * {@link https://dev.maxmind.com/minfraud/api-documentation/responses/#schema--response--risk-score-reason--multiplier-reason | the response API documentation }.
```

or something along those lines.